doc/reference/networking/net_context.rst

```diff
@@ -6,6 +6,8 @@ Networking Context
 Overview
 ********
 
+This API is being phased out and user should use :ref:`bsd_sockets_interface`
```

Comment:
it will be still here, but only for the core part, so maybe rephrase "phased out"?

Reply:
Yes, I can modify it to say that it is not meant for application use any more.